src/routes/alerts/alerts.controller.spec.ts

```diff
@@ -178,11 +178,12 @@ describe('Alerts (Unit)', () => {
         );
 
         networkService.get.mockImplementation((url) => {
+          console.log(url);
           switch (url) {
             case `${safeConfigUrl}/api/v1/chains/${chain.chainId}`:
               return Promise.resolve({ data: chain });
             case `${chain.transactionService}/api/v1/modules/${delayModifier}/safes/`:
-              return Promise.resolve({ data: { safes: [safe.address] } });
+              return Promise.resolve({ safes: [safe.address] });
```

Comment:
This change can be a bit confusing since the expected return type has been `NetworkResponse`.

I know that this is a default and not a requirement but I wonder if we should change that since this seems like an outlier 🤔

Reply:
Good catch. I missed an inconsistency between the `CacheFirstDataSource` and using the `NetworkService` directly. I've adjusted it in 98d925cac1549b16372a4d297e4e96c60f14d219 accordingly.